Date: Tue, 25 May 2021 19:32:28 +0200
Subject: [PATCH] Remove "disable addon dialog at startup" for LibreELEC.

The feature is not only not needed for LE but as well does enable the DVB
driver addons.

See https://github.com/LibreELEC/LibreELEC.tv/issues/5397 and
https://github.com/xbmc/xbmc/pull/19091
---
 xbmc/platform/linux/PlatformLinux.h | 1 -
 1 file changed, 1 deletion(-)

--- a/xbmc/platform/linux/PlatformLinux.h
+++ b/xbmc/platform/linux/PlatformLinux.h
@@ -23,7 +23,6 @@ public:
   bool InitStageOne() override;
   void DeinitStageOne() override;
 
-  bool IsConfigureAddonsAtStartupEnabled() override { return true; }
 
 private:
   std::unique_ptr<OPTIONALS::CLircContainer, OPTIONALS::delete_CLircContainer> m_lirc;
